extensions/cal-com-share-meeting-links/src/api/cal.com.ts

```diff
@@ -123,7 +123,7 @@ export interface updateBookingContent {
 }
 
 const defaultBaseUrl = "https://api.cal.com/v1/";
-const { token } = getPreferenceValues<Preferences>();
+const { token } = getPreferenceValues<PreferenceValues>();
```

Comment:
Nitpick (non-blocking): Why are we changing this? I thought raycast recommends using the types generated in _raycast-env.d.ts_ at the end of [this doc](https://developers.raycast.com/api-reference/preferences#getpreferencevalues).